docs/en/integrations/language-clients/java/client-v2.md

```diff
@@ -71,6 +113,63 @@ Major configuration parameters are defined in one scope (client or operation) an
 
 Configuration is defined during client creation. See `com.clickhouse.client.api.Client.Builder`.
 
+## Client Configuration
+
+| Configuration Method                  | Arguments                                      |  Description                                |
+|---------------------------------------|:-----------------------------------------------|:--------------------------------------------|
+| addEndpoint(String endpoint)          | - `enpoint` - URL formatted a server address.      | Adds a server endpoint to list of available servers. Currently only one endpoint is supported. |
+| addEndpoint(Protocol protocol, String host, int port, boolean secure) | - `protocol` - connection protocol `com.clickhouse.client.api.enums.Protocol#HTTP`.<br />- `host` - IP or hostname of a server.<br />- `secure` - if communication should use secure version of the protocol (HTTPs) | Adds a server endpoint to list of available servers. Currently only one endpoint is supported. |
+| setOption(String key, String value)   | - `key` - String key of an option.<br /> - `value` - String value of an option | Sets raw value of client options. Useful when reading configuration from properties files. It helps to avoid calling corresponding builder methods. | 
```

Comment:
How about
> `key` - String key of a client configuration option. 

should add a link to the list of client configuration options?

Reply:
Sounds good, but we do not have such list yet. Will work on it in 0.7.2 release.